test/unit/vdom/skip.js

```diff
@@ -0,0 +1,60 @@
+import { define, prop, props, symbols, vdom } from '../../../src/index';
+import afterMutations from '../../lib/after-mutations';
+import fixture from '../../lib/fixture';
+
+describe('vdom/skip', () => {
+  it('should skip the element children', done => {
+    const Elem = define('x-test', {
+      props: {
+        num: prop.number()
+      },
+      render() {
+        vdom.element('div', { skip: true }, () => {
+          vdom.element('span', () => {
```

Comment:
The last point there might actually render content. That said, it doesn't really matter. I updated the original description with another link to the updated bin showing how you'd do what Skate does manually.